Policies/ServiceBus/Azure_ServiceBus_DP_Use_Secure_TLS_Version.json

```diff
@@ -0,0 +1,64 @@
+{
+  "mode": "All",
+  "policyRule": {
+    "if": {
+      "allOf": [
+        {
+          "field": "type",
+          "equals": "Microsoft.ServiceBus/namespaces"
+        },
+        {
+          "not": {
```

Comment:
As this property is new, it might happens this property does not exist for older service namespaces
Lets also add null check.

Reply:
Addressed.